changelog.md

```diff
@@ -1,4 +1,5 @@
 # Future
+- [ADDED] Tests for `createSchema`, `databaseVersion`, `renameFunction`
```

Comment:
Shouldn't really make a difference for end users so no need to have a changelog entry :)